core/src/main/java/com/linecorp/armeria/common/stream/FilteredStreamMessage.java

```diff
@@ -66,7 +66,7 @@ protected FilteredStreamMessage(StreamMessage<T> delegate) {
     protected FilteredStreamMessage(StreamMessage<T> delegate, boolean withPooledObjects) {
         requireNonNull(delegate, "delegate");
         this.delegate = delegate;
-        this.withPooledObjects = withPooledObjects;
+        this.filterSupportsPooledObjects = withPooledObjects;
```

Comment:
nit: How about changing the parameter name as well?

Reply:
I thought about it, but what I felt is that for users of this constructor (subclasses), they care about withPooledObjects or not so it seems like a clear name, but for implementors of this class, it's confusing to have both that and the subscriber one and I renamed it for the sake of this implementation only. I'd lean towards keeping the API parameter as is.

Reply:
I agree. Thank you for explanation!